drivers/node/ssh/ssh.go

```diff
@@ -86,6 +87,29 @@ func (s *SSH) IsUsingSSH() bool {
 	return len(os.Getenv("TORPEDO_SSH_KEY")) > 0 || len(os.Getenv("TORPEDO_SSH_PASSWORD")) > 0
 }
 
+// WaitForValidUptime Wait for valid uptime.
+func (s *SSH) WaitForValidUptime(n node.Node) (interface{}, error) {
+	uptimeCmd := "sudo uptime -s"
+	t := func() (interface{}, bool, error) {
+		out, err := s.doCmd(n, node.ConnectionOpts{
+			Timeout:         1 * time.Minute,
+			TimeBeforeRetry: 10 * time.Second,
+		}, uptimeCmd, true)
+		return out, true, err
+	}
+	uptimecall := func() (interface{}, bool, error) {
+		out, err := task.DoRetryWithTimeout(t, 1*time.Minute, 10*time.Second)
+		if err != nil {
```

Comment:
```suggestion
		if err != nil || if len(strings.Fields(strings.TrimSpace(out.(string)))) == 0 {
```

Reply:
Changed logic and now extra function is not required.